content/webapp/components/OnThisPageAnchors/OnThisPageAnchors.tsx

```diff
@@ -7,7 +7,14 @@ import { FunctionComponent } from 'react';
 const Anchor = styled.a.attrs(() => ({
   className: font('intb', 5),
 }))`
-  color: ${props => props.theme.color('accent.green')};
+  color: ${props => props.theme.color('neutral.600')};
```

Comment:
I think I'd go darker so `neutral.700` or `black`, even

Reply:
I know we've yet to find the perfect way to style our links, one to keep in mind for the future changes maybe